packages/server/modules/workspaces/tests/integration/workspaces.graph.spec.ts

```diff
@@ -116,37 +117,176 @@ describe('Workspaces GQL CRUD', () => {
     })
 
     describe('query workspace.team', () => {
+      let largeWorkspaceApollo: TestApolloServer
+
+      const largeWorkspace: BasicTestWorkspace = {
+        id: '',
+        ownerId: '',
+        name: 'My Large Workspace',
+        description: 'A workspace with many users and roles to test pagination.'
+      }
+
+      const largeWorkspaceAdmin: BasicTestUser = {
+        id: '',
+        name: 'John A Speckle',
+        email: 'large-workspace-user-a@example.org'
+      }
+
+      before(async () => {
+        await createTestUser(largeWorkspaceAdmin)
+        await createTestWorkspace(largeWorkspace, largeWorkspaceAdmin)
+
+        largeWorkspaceApollo = await testApolloServer({
+          authUserId: largeWorkspaceAdmin.id
+        })
+
+        const workspaceMembers: [BasicTestUser, WorkspaceRoles][] = [
+          [
+            {
+              id: '',
+              name: 'John B Speckle',
+              email: 'large-workspace-user-b@example.org'
+            },
+            'workspace:admin'
+          ],
+          [
+            {
+              id: '',
+              name: 'John C Speckle',
+              email: 'large-workspace-user-c@example.org'
+            },
+            'workspace:member'
+          ],
+          [
+            {
+              id: '',
+              name: 'John D Speckle',
+              email: 'large-workspace-user-d@example.org'
+            },
+            'workspace:member'
+          ],
+          [
+            {
+              id: '',
+              name: 'John F Speckle',
+              email: 'large-workspace-user-f-1@example.org'
+            },
+            'workspace:guest'
+          ],
+          [
+            {
+              id: '',
+              name: 'John F Speckle',
+              email: 'large-workspace-user-f-2@example.org'
+            },
+            'workspace:guest'
+          ]
+        ]
+
+        for (const [user, role] of workspaceMembers) {
+          await createTestUser(user)
+          await assignToWorkspace(largeWorkspace, user, role)
+          // Overly-careful guarantee that `createdAt` values are different
+          await wait(50)
```

Comment:
50 seems too much, even wait(1) should be fine, I would think

Reply:
+1 done